diff --git a/node_modules/@types/react-native/index.d.ts b/node_modules/@types/react-native/index.d.ts
index 9f4fd74..75b858a 100644
--- a/node_modules/@types/react-native/index.d.ts
+++ b/node_modules/@types/react-native/index.d.ts
@@ -4778,6 +4778,12 @@ export interface TouchableWithoutFeedbackPropsIOS {
  * @see https://facebook.github.io/react-native/docs/touchablewithoutfeedback.html#props
  */
 export interface TouchableWithoutFeedbackProps extends TouchableWithoutFeedbackPropsIOS, AccessibilityProps {
+    /**
+     * Fix because this property was not provided OOB. Disables sound when pressing on a
+     * TouchableWithoutFeedback compoennt
+     */
+    touchSoundDisabled: boolean;
+
     /**
      * Delay in ms, from onPressIn, before onLongPress is called.
      */
