diff --git a/node_modules/react-test-renderer/cjs/react-test-renderer.development.js b/node_modules/react-test-renderer/cjs/react-test-renderer.development.js
index 51666d9..61eab69 100644
--- a/node_modules/react-test-renderer/cjs/react-test-renderer.development.js
+++ b/node_modules/react-test-renderer/cjs/react-test-renderer.development.js
@@ -15894,11 +15894,15 @@ var ReactTestRendererFiber = {
       enumerable: true,
       get: function () {
         if (root === null) {
-          throw new Error("Can't access .root on unmounted test renderer");
+          throw new Error("Can't access .root on unmounted test renderer NO ROOT");
         }
         var children = getChildren(root.current);
         if (children.length === 0) {
-          throw new Error("Can't access .root on unmounted test renderer");
+          console.log("children " + children)
+          console.log("root current " + root.current)
+          console.log("root " + Object.keys(root));
+          return wrapFiber(root.current);
+          throw new Error("Can't access .root on unmounted test renderer NO CHILDREN");
         } else if (children.length === 1) {
           // Normally, we skip the root and just give you the child.
           return children[0];
